src/lib/yupValidation.ts

```diff
@@ -87,4 +87,9 @@ export const admitCardRequirementsSchema = yup.object({
     .required("Email is required")
     .min(3, "Please enter more then 3 characters")
     .max(55, "Please enter within 55 characters"),
```

Comment:
This is confusing, please change it to:
Character limit reached, maximum allowed characters is 55.